integration-tests/tzip16-originate-wallet-having-metadata-on-IPFS-and-fetch-those.spec.ts

```diff
@@ -17,13 +17,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     let contractAddress: string;
 
-    describe(`Originating contracts made with wallet api having metadata stored at HTTPS URL using: ${rpc}`, () => {
+    describe(`Test contract origination having metadata stored at HTTPS URL through wallet api using: ${rpc}`, () => {
```

Comment:
Should be IPFS not HTTPS

Reply:
Changed to IPFS.
Also removed jest retries